osu.Game/Database/BeatmapDatabase.cs

```diff
@@ -144,7 +144,11 @@ public void Import(ArchiveReader archiveReader)
         public void Import(params string[] paths)
         {
             foreach (string p in paths)
-                Import(p);
+            {
+                //Somehow this fixes a crash when importing a beatmap twice in quick succession

```

Comment:
It fixes it because imported maps have their file deleted once they're in the database. It happens in Import(string path) (Line 117)